packages/rocketchat-channel-settings/client/startup/trackSettingsChange.js

```diff
@@ -3,8 +3,9 @@ Meteor.startup(function() {
 		Tracker.nonreactive(() => {
 			if (msg.t === 'room_changed_privacy') {
 				if (Session.get('openedRoom') === msg.rid) {
-					const type = FlowRouter.current().route.name === 'channel' ? 'c' : 'p';
-					RoomManager.close(type + FlowRouter.getParam('team') + '/' + FlowRouter.getParam('name'));
+					// ttrc logic - "Eslint - const type is assigned a value but never used"
+					// const type = FlowRouter.current().route.name === 'channel' ? 'c' : 'p';
+					RoomManager.close(`$ {type + FlowRouter.getParam('team')  }/${ FlowRouter.getParam('name') }`);
```

Comment:
You have 2 errors here. there should be no space between the $ and {, thus it's not interpreting the variables, which is why you get the "Eslint - const type is assigned a value but never used" message